spec.html

```diff
@@ -39722,6 +39750,11 @@ <h1>Agent Events Records</h1>
             <td>A List of events</td>
             <td>Events are appended to the list during evaluation.</td>
           </tr>
+          <tr>
+            <td>[[AgentSynchronizesWith]]</td>
+            <td>A Relation on Synchronize events</td>
```

Comment:
Please change this to a List.

The framework I've tried to adhere to is that the operational semantics appends events directly to Lists, and Relations are built by the axiomatic semantics from those Lists. (Also the Relations are not constructed by AOs to avoid implying that the axiomatic semantics is a step-by-step thing.)

Reply:
Done - I may submit a separate editorial PR to make the spec's use of the word "pair" slightly more consistent.